docs/source/releases/v3.2.rst

```diff
@@ -39,24 +43,61 @@ Minor changes
 
 - Added a new helper ``core.utils.is_ajax`` which replicates the logic of Django's ``HttpRequest.is_ajax``
   method that was deprecated in Django 3.1.
+  
 - Persisted ``OrderFactory.date_placed`` to the database when set, instead of just setting it on the object without saving it.
 
+- Fixed getting non public categories with ``get_categories`` by only returning browsable categories.
+
+- Reviews can now only be viewed and placed when the product is public. Before you could access the review urls if you knew them or they were indexed into a search engine.
+
+?- Remove title from shippingaddress
+?- Remove title from billing address
+
+- Change ``PartnerProductFilterMixin`` queryset to also include child products for the partner user.
+
+- Let the ``RangeDetailView`` listen to the ``OSCAR_PRODUCTS_PER_PAGE`` setting for the amount of products to be shown on the range page.
```

Comment:
Please change to say "Changed `RangeDetailView` to respect the `OSCAR_PRODUCTS_PER_PAGE` setting for pagination."